doc/build-openbsd.md

```diff
@@ -61,10 +58,17 @@ export AUTOMAKE_VERSION=1.15 # replace this with the automake version that you i
 ```
 Make sure `BDB_PREFIX` is set to the appropriate path from the above steps.
 
+To configure with wallet and Qt 5:
+```bash
+./configure --with-gui=qt5 CC=cc CXX=c++ \
+    LDFLAGS="-L${BDB_PREFIX}/lib -L/usr/X11R6/lib" \
+    CPPFLAGS="-I${BDB_PREFIX}/include -I/usr/X11R6/include"
+```
+
 To configure with wallet:
 ```bash
 ./configure --with-gui=no CC=cc CXX=c++ \
-    BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" BDB_CFLAGS="-I${BDB_PREFIX}/include"
+    LDFLAGS="-L${BDB_PREFIX}/lib" CPPFLAGS="-I${BDB_PREFIX}/include"
```

Comment:
Why switch to using LDFLAGS/CPPFLAGS instead of `BDB_LIBS`/`BDB_FLAGS` which exist specifically for the purpose of overriding BDB flags?

Reply:
My bad. I went with whatever install_db4.sh suggested. Also, when compiling with Qt 5 support, you also need to specify the paths for /usr/X11R6/{lib,include}.

Reply:
Good point, install_db4.sh should suggest using these instead.

Reply:
FWIW changing install_db4.sh was done in 311a423bd8642d4c507b2997990522715e590eca, update it here too please.